src/components/RightDockedModal.js

```diff
@@ -33,7 +33,7 @@ const RightDockedModal = memo(({
     <Modal
         type={CONST.MODAL.MODAL_TYPE.RIGHT_DOCKED}
         onClose={redirectToLastReport}
-        isVisible={currentURL === route}
+        isVisible={currentURL.includes(route)}
```

Comment:
it allows for subroutes to work since the route is `/settings` and the subroute/currentUrl could be `/settings/profile`.

Basically they would all go to the SettingsPage then that component will handle the logic on rendering the right subroute. (btw did you mean a comment in the code or on the pr? 😅 )

Reply:
Haha I meant a comment in the code itself that you're submitting
```js
// Using includes allows for subroutes of /settings to work. 
// All /settings/url subroutes would go to the SettingsPage and that page will render the right components for the subroute
isVisible={currentURL.includes(route)}
```